tools/shared.py

```diff
@@ -1077,16 +1077,19 @@ def check(value):
   return list(filter(check, values))
 
 
+SIZE_SUFFIXES = {suffix: 1024 ** i for i, suffix in enumerate(['b', 'kb', 'mb', 'gb', 'tb'])}
+
+
 def expand_byte_size_suffixes(value):
-  """Given a string with arithmetic and/or KB/MB size suffixes, such as
-  "1024*1024" or "32MB", computes how many bytes that is and returns it as an
-  integer.
   """
```

Comment:
Docstrings normally start on the same line as the opening triple quote. See https://www.python.org/dev/peps/pep-0257/